src/main/java/org/broadinstitute/hellbender/engine/TargetWalker.java

```diff
@@ -10,12 +10,14 @@
 /**
  * Super class for tools that traverse through targets.
  *
+ * Target files have the format described in {@link org.broadinstitute.hellbender.tools.exome.TargetWriter}
+ *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
 public abstract class TargetWalker extends FeatureWalker<Target> {
 
     @Argument(
-            doc = "target BED file.",
+            doc = "target file.",
```

Comment:
``Target file -- Not a BED file.  Should be formatted as a tsv with the minimum following columns, in a header: ......``